README.md

```diff
@@ -1,43 +1,81 @@
-# Secvisogram Backend
+# BSI Secvisogram CSAF Backend
 
 ![Coverage](https://raw.githubusercontent.com/secvisogram/csaf-cms-backend/badges/.github/badges/jacoco.svg)
 
+- [About The Project](#about-the-project)
+- [Getting started](#getting-started)
+- [How to use](#how-to-use)
+- [Developing](#developing)
+- [Contributing](#contributing)
+- [Dependencies](#dependencies)
+
+## About The Project
+
+TODO
+
+[(back to top)](#bsi-secvisogram-csaf-backend)
+
 ##  Getting started
 
-The configuration of the application as well as the compose file is done in 
+To run the CSAF CMS server you need the following:
+
+- [Keycloak](https://www.keycloak.org/)
+- A proxy like [oauth2-proxy](https://oauth2-proxy.github.io/oauth2-proxy/)
+- [CouchDB](https://couchdb.apache.org/)
+
+You can find an example setup for local development in the 'compose.yaml' and 
+an example configuration for Keycloak in 'keycloak/csaf-realm.json'. You 
+can take this as a starting point, but please check the documentation of the
+individual projects for a proper production setup.
+
+The [secvisogram](https://github.com/secvisogram/secvisogram) frontend is usable
+as a standalone version without this server. You can still use this standalone
+mode if the frontend is not behind the proxy, like in the development setup.
+In this case the login is only required to manage documents on the server 
+or validate against the [validator service](https://github.com/secvisogram/csaf-validator-service).
+
+[(back to top)](#bsi-secvisogram-csaf-backend)
+
+## How to use
+
+Please have a look at the [API documentation](https://secvisogram.github.io/csaf-cms-backend/) on how to use this application.
+
+[(back to top)](#bsi-secvisogram-csaf-backend)
+
+## Developing
+
+The configuration of the application as well as the compose file is done in
 a local **.env** file. To start, simply copy the **.env.example** file.
-If you want different passwords, database names or ports you can change them 
-here.
+If you want different passwords, database names or ports you can change them
+here. Please note that the following setup is for development purposes only 
+and should not be used in production.
 
-- run `docker-compose up`
-- After Keycloak is up, open a second terminal window and run 
-  `docker-compose up csaf-keycloak-cli` to import a realm with all the users 
+- run `docker compose up`
+- After Keycloak is up, open a second terminal window and run
+  `docker compose up csaf-keycloak-cli` to import a realm with all the users
```

Comment:
I don't think these changes are correct.

```
$ docker compose up
docker: 'compose' is not a docker command.
```

Or is this a new feature and I'm lagging behind?

Reply:
'docker compose' is https://github.com/docker/compose/tree/v2 that moves the old 'docker-compose' functionality to docker as part of the Docker CLI. Depending on the Linux version you are using it could be that you need to install this as a plugin. (Or just keep using 'docker-compose' for now)

Reply:
I see. Thanks for clarifying!